File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    var result = 0;
    for(let i =0; i<babbling.length; i++){
        let str = babbling[i];
        let x = 0;
        while(str.length > 0){
            if(str.slice(0,3) === "aya"){
                str = str.slice(3)
            }
            else if(str.slice(0,2) === "ye"){
                str = str.slice(2)
            }
            else if(str.slice(0,3) === "woo"){
                str = str.slice(3)
            }
            else if(str.slice(0,2) === "ma"){
                str = str.slice(2)
            }
            else{
                x = 1;
                break;
            }
        }
        if(x === 0){
            result++;
        }
                
    }
    return result;
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    bab_list = ["aya", "ye", "woo", "ma"]
    ans = 0
    
    for i in babbling:
        for j in bab_list:
            if j in i:
                i = i.replace(j, ' ')
        
        if i.strip() == '':
            ans += 1
    
    return ans
        
    
    ''